src/main/scala/devices/gpio/IOF.scala

```diff
@@ -63,3 +63,18 @@ object BasePinToIOF {
   }
 }
 
+object IOFPinFromInputPort {
```

Comment:
for naming consistency with the above, shouldn't this be called InputPortToIOF?

Also can you add some code comments similar to the above function for what this is doing?

Reply:
I already fix this, but not yet pushed.